lib/insights_cloud/async/connector_playbook_execution_reporter_task.rb

```diff
@@ -0,0 +1,192 @@
+module InsightsCloud
+  module Async
+    class ConnectorPlaybookExecutionReporterTask < ::Actions::EntryAction
+      include Dynflow::Action::Polling
+      include ForemanRhCloud::CloudAuth
+
+      def self.subscribe
+        Actions::RemoteExecution::RunHostsJob
+      end
+
+      def self.connector_feature_id
+        @connector_feature_id ||= RemoteExecutionFeature.feature!(:rh_cloud_connector_run_playbook).id
+      end
+
+      def plan(job_invocation)
+        return unless connector_playbook_job?(job_invocation)
+
+        @job_invocation = job_invocation
```

Comment:
Why though?

Reply:
To cache the job invocation instance. Otherwise, the code will re-fetch the instance that I already have.